existing_lints/tokio_time_interval/src/tokio_time_interval.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Nord Security
+use clippy_utils::diagnostics::span_lint_and_help;
+use rustc_hir::{Expr, ExprKind, Item, ItemId};
+use rustc_lint::{LateContext, LateLintPass};
+use rustc_middle::ty::TyKind;
+use rustc_middle::ty::VariantDef;
+use rustc_session::{declare_lint, impl_lint_pass};
+use rustc_span::{def_id::DefId, sym};
+use std::collections::HashSet;
+
+// More info on this macro can be found:
+// https://doc.rust-lang.org/nightly/nightly-rustc/rustc_session/macro.declare_lint.html
+// https://rustc-dev-guide.rust-lang.org/diagnostics.html
+declare_lint! {
+    /// **What it does:** Checks for use of blocking tokio::time::interval(_at) calls
+    ///
+    /// **Why is this bad?** This might cause generation of many events if used incorrectly
+    ///
+    /// **Example:**
+    ///
+    /// If there is some periodic action that is triggered by the resulting Interval instance,
+    /// after a long period of sleep it will be triggered 'sleep_duration/interval_period' as fast
+    /// as possible until it is caught up in time to where it should be. Most of the time this is
+    /// not a desired behaviour. This default behaviour can be controlled with the 'set_missed_tick_behavior'
+    /// setter. In libtelio there is already a wrapper that returns correctly configured interval.
+    pub TOKIO_TIME_INTERVAL,
+    Warn,
+    "description goes here"
```

Comment:
Maybe a proper description would be nice?

Reply:
Done 😄